hibernate4/src/main/java/com/fasterxml/jackson/datatype/hibernate4/Hibernate4Module.java

```diff
@@ -28,21 +28,21 @@
          * Default value is true.
          */
         USE_TRANSIENT_ANNOTATION(true),
-        
+
 	    /**
 	     * If FORCE_LAZY_LOADING is false, this feature serializes uninitialized lazy loading proxies as
-	     * <code>{"identifierName":"identifierValue"}</code> rather than <code>null</code>. 
+	     * <code>{"identifierName":"identifierValue"}</code> rather than <code>null</code>.
 	     * <p>
          * Default value is false.
 	     * <p>
-	     * Note that the name of the identifier property can only be determined if 
+	     * Note that the name of the identifier property can only be determined if
 	     * <ul>
 	     * <li>the {@link Mapping} is provided to the Hibernate4Module, or </li>
-	     * <li>the persistence context that loaded the proxy has not yet been closed, or</li> 
+	     * <li>the persistence context that loaded the proxy has not yet been closed, or</li>
 	     * <li>the id property is mapped with property access (for instance because the {@code @Id}
 	     * annotation is applied to a method rather than a field)</li>
 	     * </ul>
-	     * Otherwise, the entity name will be used instead. 
+	     * Otherwise, the entity name will be used instead.
```

Comment:
Can you add "@since 2.12" javadoc tag at the end of this comment (and similar for hibernate5)